docs/manual/developer_guide.adoc

```diff
@@ -227,6 +227,208 @@ The maintainer should check Bohdi packages for that update for positive/negative
+
+|```tests```
+| Contains the test suite for content validation and testing.
+|===
+
+The remaining directories such as `fedora`, `rhel7`, etc. are product
+directories.
+
+===== Important Top Level File Descriptions
+
+|===
+|File |Description
+
+|```BUILD.md```
+| Contains the content build instructions
+
+
+|```CMakeLists.txt```
+| Top-level CMake build configuration file
+
+|```Contributors.md```
+| *DO NOT MANUALLY EDIT* script-generated file
+
+|```Contributors.xml```
+| *DO NOT MANUALLY EDIT* script-generated file
+
+|```DISCLAIMER```
+| Disclaimer for usage of content
+
+|```Dockerfile```
+| CentOS7 Docker build file
+
+|```LICENSE```
+| Content license
+
+|```oval.config.in```
+| _Deprecated in future releases:_
+Build configuration for mapping product
+version to correspond `multi_platform` product tags
+
+|```README.md```
+| Project README file
+
+|===
+
+==== Product Structure/Layout
+
+When creating a new product, use the guidelines below for the directory layout:
+
+* *Do not* use capital letters
+* If product versions are required, use major versions only. For example,
+`rhel7`, `ubuntu16`, etc.
+* If the content to be produced does not matter on versions, *do not* add version
+numbers. For example: `fedora`, `firefox`, etc.
+* In addition, use only a maxdepth of 3 directories.
+
+Following these guidelines help with the usability and browsability of
+using and navigating the content.
+
+For example:
+[source,bash]
+----
+$ tree -d rhel7
+
+rhel7
+├── checks
+│   └── oval
+├── cpe
+├── fixes
+│   ├── ansible
+│   └── bash
+├── kickstart
+├── overlays
+├── profiles
+├── templates
+│   ├── csv
+├── transforms
+└── utils
+
+13 directories
+----
+
+===== Product Level Directory Descriptions
+
+|===
+|Directory |Description
+
+|```checks```
+|```[red]#Required#``` Contains content such as OVAL to check whether or not a
+system is configured correctly to meet government or commercial compliance
+standards. Can contain the following directories: ```oval```
+
+|```cpe```
+|```[red]#Required#``` Contains the Common Platform Enumeration (CPE) product
+identifier that is provided from link:https://nvd.nist.gov/products/cpe[NIST].
+
+|```fixes```
+|```[red]#Required#``` Contains scripts in various languages that fixes
+configuration to meet government or commercial compliance standards. Can contain
+the following directories: ```bash``` ```ansible``` ```puppet``` ```anaconda```
+
+|```kickstart```
+|```[red]#Optional#``` Contains product kickstart or build files to be used in
+testing, development, or production (not recommended) of compliance content.
+
+|```overlays```
+|```[red]#Required#``` Contains overlay files for specific standards
+organizations such as NIST, DISA STIG, PCI-DSS, etc.
+
+|```profiles```
+|```[red]#Required#``` Contains profiles that are created and tailored to meet
+government or commercial compliance standards.
+
+|```templates```
+|```[red]#Required#``` Can contain the following directories: ```csv```
```

Comment:
It would be nice to mention the `oval_5.11_templates` directory.

Reply:
The reason this, `oval_5.11`, and `utils` are not mentioned is because it is going to be removed.